giraffe/src/utils/annotationData.test.ts

```diff
@@ -0,0 +1,55 @@
+import {sortDistances} from './annotationData'
+
+describe('annotationData utils', () => {
+  describe('sort distances', () => {
+    it('handles empty data', () => {
+      const empty = []
+      empty.sort(sortDistances)
+      expect(empty).toEqual([])
+    })
+    it('a point annotation within the overlap range should be the minimum when there is an overlapping range annotation', () => {
```

Comment:
`it("should read like a sentence", ...)`

`it("does x and y should be the result", ...)`

Reply:
The `it` should flow right into the test description like the examples above.